web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2017 Therp BV, ACSONE SA/NV
+ *  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web.web_ir_actions_act_window_message', function(require)
+{
+    "use strict";
+
+    var ActionManager = require('web.ActionManager'),
+        core = require('web.core'),
+        Dialog = require('web.Dialog');
+
+    var _t = core._t;
+
+    ActionManager.include({
+        _handleAction: function (action, options) {
+            if (action.type === 'ir.actions.act_window.message') {
+                return this.ir_actions_act_window_message(action, options);
+            }
+            return this._super.apply(this, arguments);
+        },
+        ir_actions_act_window_message: function(action, options)
+        {
+            var self = this,
+                buttons = [];
+
+            if(action.close_button_title !== false)
+            {
+                buttons.push({
+                    text: action.close_button_title || _t('Close'),
+                    click: function() {
+                        // refresh the view before closing the dialog
+                        var controller = self.getCurrentController();
+                        if (controller && controller.widget) {
+                            controller.widget.reload();
+                        }
+                        dialog.close()
+                    },
+                    classes: 'btn-default',
+                })
+            }
+
+            var is_html = action.is_html_message === true;
+            var content_properties = {};
+
+            if (is_html) {
+                content_properties = {
+                    html: action.message,
+                };
+            } else {
+                content_properties = {
+                    text: action.message,
+                    css: {
+                        'white-space': 'pre-line',
+                    }
+                };
+            }
+
+            var dialog = new Dialog(
+                this,
+                _.extend(
+                    {
+                        size: 'medium',
+                        title: action.title,
+                        $content: $('<div>', content_properties),
+                        buttons: buttons.concat(
+                            this.ir_actions_act_window_message_get_buttons(
+                                action, function() { dialog.close() })
+                        ),
+                    },
+                    options)
+            )
+            dialog.open();
+            return $.when();
```

Comment:
it's probably better if we return https://github.com/OCA/OCB/blob/12.0/addons/web/static/src/js/core/dialog.js#L56 here

Reply:
Done!

Reply:
thanks, but I don't see this in the diff. I'd expect something like `return dialog.open()._opened`

Reply:
Sorry, I was wrong before.